### PR TITLE
Remove un-needed puts from configuration_file.rb

### PR DESCRIPTION
**Pull request by joshdholtz**
Fixes https://github.com/fastlane/fastlane/pull/11946/files#r192949219

## What
- `puts` was not actually needed so it has been removed now

### CODE REVIEW

File: fastlane_core/lib/fastlane_core/configuration/configuration_file.rb [hunk 1]
```diff
@@ -43,7 +43,7 @@ def initialize(config, path, block_for_missing, skip_printing_values = false)
 
       begin
         # rubocop:disable Security/Eval
-        puts(eval(content)) # this is okay in this case
```
Comment (taquitos) [inline fastlane_core/lib/fastlane_core/configuration/configuration_file.rb]: Huh, any idea why it was there in the first place?

**Review by taquitos — APPROVED** [review]
🎈:goat:

File: fastlane_core/lib/fastlane_core/configuration/configuration_file.rb (hunk 1 above)
Comment (joshdholtz) [reply fastlane_core/lib/fastlane_core/configuration/configuration_file.rb]: Something I missed when reviewing #11946 🙃

Comment (revolter) [reply fastlane_core/lib/fastlane_core/configuration/configuration_file.rb]: @taquitos It was added by https://github.com/matsune/fastlane/commit/7d12cdabc8850225daa44a0a66477e94d1b0dd78

Comment (revolter) [reply fastlane_core/lib/fastlane_core/configuration/configuration_file.rb]: I'm guessing it was added for debugging purposes and was left there by mistake.

Comment (taquitos) [reply fastlane_core/lib/fastlane_core/configuration/configuration_file.rb]: Oh, that makes sense. I suspected that to be the case. Thanks for checking!

**Pull request merged**